resources/dynamic-das/Makefile

```diff
@@ -1,17 +1,15 @@
 .PHONY: build
 
-VERSION = 1.0.0
+VERSION = 1.0.1
 
 build:
 	echo "Build amd64"
 	docker buildx build --platform=linux/amd64 -t etendo/dynamic-das:$(VERSION)-amd64 .
```

Comment:
This code where would it run? remember that if it runs on a linux with amd64 it will not be able to build the image for arm64, unless it is a Mac.

Reply:
This will be made in task ETP-246